libraries/AP_OpenDroneID/AP_OpenDroneID.cpp

```diff
@@ -106,11 +108,6 @@ bool AP_OpenDroneID::pre_arm_check(char* failmsg, uint8_t failmsg_len)
         return true;
     }
 
-    if (pkt_basic_id.id_type == MAV_ODID_ID_TYPE_NONE) {
```

Comment:
This was originally added to confirm the receipt of the `BASIC_ID` message and that a field had been filled.

We could switch to checking a different field?

Or I'm guessing since `OPEN_DRONE_ID_ARM_STATUS` now exists you think the check here isn't needed and can be annoying?